packages/onboarding/src/utils/flows.ts

```diff
@@ -6,6 +6,7 @@ export const LINK_IN_BIO_POST_SETUP_FLOW = 'link-in-bio-post-setup';
 export const VIDEOPRESS_FLOW = 'videopress';
 export const IMPORT_FOCUSED_FLOW = 'import-focused';
 export const ECOMMERCE_FLOW = 'ecommerce';
+export const ECOMMERCE_TRIAL_FLOW = 'ecommerce-trial';
```

Comment:
I think we should more clearly differentiate this flow as one specific to WooExpress.